### PR TITLE
TNL-6389: Adds in triggers for ORA2 Jobs

### PR DESCRIPTION
**Pull request by jlajoie**
## [TNL-6389](https://openedx.atlassian.net/browse/TNL-6389)

Adds in a daily timer trigger as well as a Git material to trigger on pushes to edx-ora2 master. Additionally sets defaults for the dns and name environment variables for the timer material of the pipeline to execute with.

FYI: @staubina @cahrens 

- [ ] Rebase and squash commits

### CODE REVIEW

File: edxpipelines/pipelines/build_ora2_sandbox.py [hunk 1]
```diff
@@ -28,14 +29,24 @@ def install_pipelines(configurator, config, env_configs):  # pylint: disable=unu
...
 
     pipeline.ensure_material(TUBULAR())
 
+    pipeline.ensure_material(
+        GitMaterial(
+            url='https://github.com/edx/edx-ora2',
+            branch='master',
+            polling=True,
+            destination_directory='edx-ora2'
+        )
```
Comment (doctoryes) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: Could you instead define this material in `materials.py` and reference that definition, as `TUBULAR()` is above?

File: edxpipelines/pipelines/build_ora2_sandbox.py [hunk 2]
```diff
@@ -28,14 +29,24 @@ def install_pipelines(configurator, config, env_configs):  # pylint: disable=unu
     """
 
     pipeline = configurator.ensure_pipeline_group(constants.ORA2_PIPELINE_GROUP_NAME) \
-                           .ensure_replacement_of_pipeline(constants.BUILD_ORA2_SANDBOX_PIPELINE_NAME)
+                           .ensure_replacement_of_pipeline(constants.BUILD_ORA2_SANDBOX_PIPELINE_NAME) \
+                           .set_timer('30 9 * * * *')
```
Comment (doctoryes) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: I haven't verified this at all - but just be aware that GoCD uses the Quartz format for its cron config:
http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html

File: edxpipelines/pipelines/build_ora2_sandbox.py (hunk 1 above)
Comment (jlajoie) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: Can do

File: edxpipelines/pipelines/build_ora2_sandbox.py (hunk 2 above)
Comment (jlajoie) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: Ah, understood, can update to be reflective of what we're actually intending which is an early morning run once a day.

Comment (macdiesel) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: The current format would run this every hour at 9 minutes and 30 seconds past the hour.

File: edxpipelines/pipelines/build_ora2_sandbox.py [hunk 3]
```diff
@@ -28,14 +28,17 @@ def install_pipelines(configurator, config, env_configs):  # pylint: disable=unu
     """
 
     pipeline = configurator.ensure_pipeline_group(constants.ORA2_PIPELINE_GROUP_NAME) \
-                           .ensure_replacement_of_pipeline(constants.BUILD_ORA2_SANDBOX_PIPELINE_NAME)
+                           .ensure_replacement_of_pipeline(constants.BUILD_ORA2_SANDBOX_PIPELINE_NAME) \
+                           .set_timer('0 30 9 * * ?')
 
     pipeline.ensure_material(TUBULAR())
 
+    pipeline.ensure_material(EDX_ORA2())
```
Comment (doctoryes) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: Nit:
```
for material in (TUBULAR, EDX_ORA2):
    pipeline.ensure_material(material())
```

Comment (jlajoie) [inline edxpipelines/pipelines/build_ora2_sandbox.py]: Reasonable enough, will update

**Pull request merged**